src/index.html

```diff
@@ -9,6 +9,27 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- LOGO -->
+    <header class="header">
+      <div class="header__content">
+        <a href="#" class="logo"><img src="./images/logo.png" alt="img"></a>
+
+
+        <!-- NAVIGATION MENU -->
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item"><a href="/" class="nav__link is-active">Apple</a></li>
```

Comment:
![image](https://github.com/mate-academy/layout_moyo-header/assets/96265844/4fe30467-329a-48d3-b324-c109226ee77f)

Reply:
Hello, should the class "is-active" be used for all items in the navigation when selected or only for the first one?